integration/resources/templates/single/basic_canary_with_tags.yaml

```diff
@@ -33,7 +35,7 @@ Resources:
               - s3:PutObject
               - s3:GetBucketLocation
             Resource:
-              - Fn::Sub: arn:aws:s3:::${ResultsBucket}/*
+              - Fn::Sub: arn:aws:s3:::${SyntheticsCanaryArtifactBucket}/*
```

Comment:
Where is the value of `SyntheticsCanaryArtifactBucket` coming from? I don't see the parameter or resources with such name

Reply:
It'll be the logical id of the artifact bucket thats automatically generated (logical id of canary + "ArtifactBucket"). Since this PR doesn't automatically generate the roles I had to hardcode in the logical id of the bucket to get the tests running but in the PR with role creation implemented it won't be hardcoded